### PR TITLE
[Overflow-450] Feature/refactor navbar

### PR DESCRIPTION
**Pull request by jescabillas**
## Backlog Link
https://framgiaph.backlog.com/view/SUN_OVERFLOW-450

## Commands
None

## Pre-conditions
None

## Expected Output
- Follow the redesign ui in figma for navbar
- Applied animation in app logo

## Notes
None

## Screenshots
image

### CODE REVIEW

**Comment by jasonchuacodes** [comment]
image

Please double check the following values:
- [x] shadow values - follow figma 
- [x] align items center
- [x] add hover state - follow figma
- [x] double check topbar height

**Review by IanUrriza10 — APPROVED** [review]

**Review by IanUrriza10 — DISMISSED** [review]

**Review by IanUrriza10 — DISMISSED** [review]

File: frontend/components/organisms/Navbar/index.tsx [hunk 1]
```diff
@@ -57,16 +57,32 @@ const Navbar = (): JSX.Element => {
...
+                        <Image
+                            height="100"
+                            width="100"
+                            alt="Sun Bear Logo"
+                            src="/images/sun_logo.png"
+                            className="h-8 w-8 transition-all duration-300 group-hover:h-9 group-hover:w-9"
+                        />
                     </div>
+                    <div className="relative z-10 flex flex-col">
+                        <span className="absolute -top-[19px] text-[20px] font-semibold text-primary-red transition-all duration-500 group-hover:text-white">
```
Comment (nicoleamber) [inline frontend/components/organisms/Navbar/index.tsx]:
```suggestion
                        <span className="absolute -top-[19px] text-xl font-semibold text-primary-red transition-all duration-500 group-hover:text-white">
```

Please apply to all related.
reference: https://tailwindcss.com/docs/font-size

**Review by nicoleamber — CHANGES_REQUESTED** [review]

File: frontend/components/organisms/Navbar/index.tsx (hunk 1 above)
Comment (jescabillas) [reply frontend/components/organisms/Navbar/index.tsx]: Applied

**Review by kenorenzsun — DISMISSED** [review]

**Review by kenorenzsun — DISMISSED** [review]

**Review by kenorenzsun — DISMISSED** [review]

**Review by nicoleamber — DISMISSED** [review]

**Comment by nicoleamber** [comment]
## QA Feedback
* Set border width to 1px and color to Neutral/200 [#E9E2E2]

image

**Comment by jescabillas** [comment]
> ## QA Feedback
> * Set border width to 1px and color to Neutral/200 [#E9E2E2]
> 
> image

Applied changes please recheck

**Comment by nicoleamber** [comment]
## Approved PR
-  All displayed the same as the Figma design
- Can hover and click on Logo to redirect me to the landing page
- Can hover and click on Notification Bell to show the notification dropdown
- Can scroll and click (redirect and mark as read) on notification items
- Can click on User Icon to show the dropdown menu for Profile and Logout
- Can click on Profile to redirect me to my Profile
- Can click on Logout to redirect me to the Login Page

## Proof
<details>

[Screencast 2023-04-20 16-47-10.webm](https://user-images.githubusercontent.com/116238730/233313465-2e2aa0c3-3d83-4c25-83eb-0d961fb53d4c.webm)

</details>

-----
- Total Test time:  0.5 hr.
- Total Comments: 1
- Total Iterations: 1

**Review by nicoleamber — APPROVED** [review]

**Review by IanUrriza10 — APPROVED** [review]

**Review by kenorenzsun — APPROVED** [review]

**Pull request merged**